test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -165,6 +168,59 @@ func describeQatDpdkPlugin() {
 			ginkgo.It("does nothing", func() {})
 		})
 	})
+
+	ginkgo.Context("When a QAT device goes unresponsive", func() {
+		ginkgo.When("QAT's auto-reset is off", func() {
+			ginkgo.BeforeEach(func() {
+				ginkgo.By("creating a configMap before plugin gets deployed")
+				e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "create", "configmap", "--from-literal", "qat.conf=$'ServiceEnabled=dc\nAutoresetEnabled=off", "qat-config")
+
+				ginkgo.By("setting resourceName for dc services")
+				resourceName = dcResource
+			})
+
+			ginkgo.It("checks if the heartbeat is read correctly [Functionality:heartbeat]", func(ctx context.Context) {
```

Comment:
IMO, these descriptions are still not very good. what "read correctly" means for a test-case? it talks about heartbeat and next it checks "there is zero resource".  "it checks unhealthy status is reported by waiting node resources become zero" or something like that.

Reply:
Thank you.! I think this kind of works is always tricky to me.. It looks much clearer now.!